src/pages/api/preview/[[...path]].ts

```diff
@@ -5,7 +5,9 @@ import serverConfig from "../../../config/server";
 import errorReporter from "../../../common-lib/error-reporter";
 import OakError from "../../../errors/OakError";
 
-export const slugStringSchema = z.string().regex(/^\w+(?:[-_]\w+)*$/);
+export const slugStringSchema = z
+  .string()
+  .regex(/^[a-zA-Z0-9]+(?:[-_][a-zA-Z0-9]+)*$/);
```

Comment:
`\w` is equivalent to `[a-zA-Z0-9_]`, so the difference here is that the character group no longer includes `_`, which I think is fine for slugs. Technically the old version would have passed for a slug starting with `_`, but I am not sure that would have been correct